protocol/socketio.go

```diff
@@ -64,15 +66,15 @@ func Encode(msg *Message) (string, error) {
 	}
 
 	if msg.Type == MessageTypeAckResponse {
-		return result + "[" + msg.Args + "]", nil
+		return result + ",[" + msg.Args + "]", nil
```

Comment:
similarly, would be good to have a protocol reference here.

Reply:
Could not find any specific protocol doc. Also guessed from the message logged by Python socketio client.